catkit/hardware/iris_ao/segmented_dm_command.py

```diff
@@ -9,7 +9,8 @@
     iris_command = segmented_dm_command.load_command(command, dm_config_id,
                                                      wavelength,
                                                      testbed_config_id,
-                                                     apply_flat_map=True)
+                                                     apply_flat_map=True,
+                                                     filename_flat=FLAT_COMMAND_FILE)
```

Comment:
Since you don't define `FLAT_COMMAND_FILE` here I recommend you just create a dummy string to make it clear what you mean.